sbt-plugin/src/main/scala/org/scalajs/sbtplugin/LinkerImpl.scala

```diff
@@ -30,7 +30,7 @@ import org.scalajs.linker.interface._
 trait LinkerImpl {
   def clearableLinker(cfg: StandardConfig): ClearableLinker
 
-  def irFileCache(): IRFileCache
+  def irFileCache(cfg: IRFileCacheConfig): IRFileCache
```

Comment:
This is debatable, but IMO we should preserve existing methods when possible, even in this API. It is an unstable API because we need the ability to *add* methods (or overloads) even though it is an open trait, so we allow ourselves to break classes that *extend* `LinkerImpl`. However, we should still make a decent effort at not breaking *users* of a `LinkerImpl`, so we should not *remove* methods unless necessary.

Therefore, I suggest adding this new overload without removing the 0-arg one.

Reply:
Hmmm... But then we should make it a final default method passing a default config?

Reply:
Yes, it's probably necessary to make it `final` to prevent existing subclasses that overrode it from still linking, since otherwise their override would never be called anymore.